discovery-provider/src/__init__.py

```diff
@@ -285,7 +285,9 @@ def configure_celery(flask_app, celery, test_config=None):
         imports=["src.tasks.index", "src.tasks.index_blacklist",
                  "src.tasks.index_plays", "src.tasks.index_metrics",
                  "src.tasks.index_materialized_views",
-                 "src.tasks.index_network_peers", "src.tasks.index_trending"],
+                 "src.tasks.index_network_peers", "src.tasks.index_trending",
+                 "src.tasks.cache_user_balance"
```

Comment:
i might call this index_user_balance since it's kind-of indexing and it might grow beyond the scope of what we have here. feel free to not change if u like this tho

Reply:
not a serious pref but i actually like 'cache_user_balance' since that is more accurate to what the activity actually is